subsys/bluetooth/shell/bt.c

```diff
@@ -2032,6 +2035,16 @@ static bool adv_param_parse(size_t argc, char *argv[],
 
 			param->peer = &addr;
 			argn += 2;
+		} else if (!strcmp(arg, "interval")) {
+			if ((argn + 2) >= argc) {
+				return false;
+			}
+			param->interval_min = strtol(argv[argn + 1], NULL, 16);
+			param->interval_max = strtol(argv[argn + 2], NULL, 16);
```

Comment:
I suggest adding in a parse error message. It does not have to be too precise, but something to draw attention to the 'interval' part of the command.

```suggestion
			err = 0;
			param->interval_min = shell_strtoul(argv[argn + 1], 16, &err);
			param->interval_max = shell_strtoul(argv[argn + 2], 16, &err);
			if (err) {
				shell_error(sh, "Bad arguments to 'interval'");
				return false;
			}
```